apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V62__create_rda_schema.sql

```diff
@@ -0,0 +1,344 @@
+create schema if not exists rda;
```

Comment:
this all looks fine to me but a comment at the top here explaining that this is being done to change the name of the schema, tables, and columns and that a future PR will drop the original schema, and that the data will not be migrated since we are just starting over (if that is correct, I cannot remember).

Reply:
I have added the comment as requested.